README.md

```diff
@@ -14,12 +14,12 @@
   <br>
   <img alt="Logo" src="fastlane/metadata/android/en-US/images/icon.png" width="100">
   <br>
-  <b>openHAB Android Client</b>
+  <b>openHAB client for Android</b>
 </p>
 
 ## Introduction
 
-openHAB Android application is a native client for openHAB. It uses REST API of openHAB to render sitemaps of your openHAB.
+This app is a native client for openHAB. It uses REST API of openHAB to render sitemaps of your openHAB.
```

Comment:
*the REST API

Reply:
Also I think I said that before: A normal user shouldn't be confronted with this API thingy he doesn't know.

Reply:
`This app is a client for openHAB for Android rendering your sitemaps.`?

Reply:
'This app is a native client for the openHAB home automation system which allows easy [or quick] access to your sitemaps'?

Reply:
In #831